dist/clouduploader.rb

```diff
@@ -73,14 +74,19 @@ def upload_image_to_ec2(image, data)
   ) do |_stdin, stdout_stderr, _wait_thr|
     while line = stdout_stderr.gets
       STDOUT.write(line)
+      write_result($1, jobid) if line =~ /Created\simage:\s(.+)/
```

Comment:
I would adapt the regex a little bit:

- Include beginning and end of line matcher ``^`` ``$``
- Match only character, digits and dash ``([\w-]+)``, otherwise it will e.g. include whitespace
- [nitpick] Match for ``ami`` -> ``(ami-[\w]+)``

```
/^Created\simage:\s([\w-]+)$/
```

Reply:
Didn't know the exact format for an AMI, so I will adapt the regexp